preact/index.d.ts

```diff
@@ -71,10 +71,10 @@ export function useStoreon<State extends object = {}, EventsMap = any>(
  * @param context User's owned React context
  * @returns useStoreon hooks that attatched to User's React context
  */
-export function customContext(context: Context<StoreonStore>): <
+export function customContext<
   State extends object = {},
   EventsMap = any
->(...keys: (keyof State)[]) => useStoreon.StoreData<State, EventsMap>
+>(context: Context<StoreonStore<State, EventsMap>>): (...keys: (keyof State)[]) => useStoreon.StoreData<State, EventsMap>
```

Comment:
Can you keep 80 column limit here?

Reply:
Maybe add new line like next commit would be ok?

Reply:
Perfect 👍